generated/shared-autoconfig-properties/src/main/java/com/google/cloud/spring/shared/SharedProperties.java

```diff
@@ -0,0 +1,19 @@
+package com.google.cloud.spring.shared;
+
+import com.google.cloud.spring.core.Credentials;
+import com.google.cloud.spring.core.CredentialsSupplier;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+
+@ConfigurationProperties("spring.auto.shared")
```

Comment:
End users are likely to have other modules that are not GCP in their applications. Typically, prefix starting with `spring` is for properties that are managed by Spring; it would be good if this prefix included "gcp" or equivalent "google.cloud" somewhere.

Reply:
Changed to `com.google.cloud.global`